### PR TITLE
Adding gcl dataset to settings

### PR DESCRIPTION
**Pull request by hiteshk-gc**
Adding a the gcl (GC Logger) dataset to the list of datasets after uploading to paperspace

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -25,13 +25,18 @@ fi
 export NUM_AVAILABLE_IPU=${IPU_ARG}
 export GRAPHCORE_POD_TYPE="pod${IPU_ARG}"
 
+# Logger specific vars
+export TIER_TYPE=$(python .gradient/check_tier.py)
+export FIREHOSE_STREAM_NAME="paperspacenotebook_production"
+
```
Comment (payoto) [inline setup.sh:31]: Maybe group the three variables? It'll make it easier to port to other repos

**Review by payoto — APPROVED** [review]
LGTM

File: setup.sh (hunk 1 above)
Comment (hiteshk-gc) [reply setup.sh:31]: What do you mean?

**Comment by hiteshk-gc** [comment]
Thanks, dataset version has been updated

File: setup.sh (hunk 1 above)
Comment (hiteshk-gc) [reply setup.sh]: Ah right, I didnt see the third one down below, thanks!

Comment (payoto) [reply setup.sh]: I changed it

**Comment by payoto** [comment]
Tested it works

**Pull request merged**